gems/quilt_rails/CHANGELOG.md

```diff
@@ -7,9 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 <!-- ## [Unreleased] -->
 
+## [1.12.1] - 2020-05-13
```

Comment:
We usually do the version bumps in the changelogs directly on master unless you plan to deploy this right after. Given that this is just for the gem, this should be ok as is.

Reply:
So for future PRs, would I leave the changelog untouched, or just omit the version bump? I guess the latter would break the lint rule that's set up

Reply:
You omit the version header changes. You add to changelog entries (the line items). The lint rule in quilt runs when we run the release on master.